ArcadiaParties-SPA/src/app/app.component.html

```diff
@@ -1,3 +1,2 @@
-<h1>WELCOME</h1>
-<h1> Title is: {{ title$ | async }} </h1>
- 
\ No newline at end of file
+<app-header [title]=" title$ | async "></app-header>
```

Comment:
As header will do much more than just show the page title, it makes sense to encapsulate all things about header inside it.

Reply:
Encapsulated prop. Now header takes data from state.